tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -114,21 +119,9 @@ def single_packet_loop(
             trace_vpacket_volley(
                 r_packet, vpacket_collection, numba_model, numba_plasma
             )
-        if mc_tracker.DEBUG_MODE:
-            r_packet_track_nu.append(r_packet.nu)
-            r_packet_track_mu.append(r_packet.mu)
-            r_packet_track_r.append(r_packet.r)
-            r_packet_track_interaction.append(interaction_type)
-            r_packet_track_distance.append(distance)
-
-    if mc_tracker.DEBUG_MODE:
-        return (
-            r_packet_track_nu,
-            r_packet_track_mu,
-            r_packet_track_r,
-            r_packet_track_interaction,
-            r_packet_track_distance,
-        )
+
+            if montecarlo_configuration.RPACKET_TRACKING:
```

Comment:
Why not just have the r_packet track at the end of all of the conditionals?

Reply:
Yes, this seems to be more logical. I will move the tracking after the conditionals :)